### PR TITLE
add get stream API + unit tests #102

### PR DESCRIPTION
**Pull request by msach22**

### CODE REVIEW

File: OpenTokTest/OpenTokTest.cs [hunk 1]
```diff
@@ -662,6 +662,32 @@ public void DeleteArchiveTest()
                 It.IsAny<Dictionary<string, object>>()), Times.Once());
         }
 
+        [Fact]
+        public void GetStreamTestFromOpenTokInstance()
```
Comment (JayTokBox) [inline OpenTokTest/OpenTokTest.cs]: How do we handle api request failures?

File: OpenTok/OpenTok.cs [hunk 2]
```diff
@@ -363,5 +363,19 @@ public void DeleteArchive(string archiveId)
...
+
+        /**
+         * Gets a Stream object for the given stream ID.
+         *
+         * @param streamId The stream ID.
+         * @return The Stream object.
+        */
+        public Stream GetStream(string sessionId, string streamId)
+        {
+            string url = string.Format("v2/project/{0}/session/{1}/stream/{2}", this.ApiKey, sessionId, streamId);
```
[the remark this answers is not in the document]
Comment (msach22) [reply OpenTok/OpenTok.cs]: Nope, that will be added as a separate PR

File: OpenTokTest/OpenTokTest.cs (hunk 1 above)
Comment (msach22) [reply OpenTokTest/OpenTokTest.cs]: The .NET SDK handles all the requests here: https://github.com/opentok/Opentok-.NET-SDK/blob/master/OpenTok/Util/HttpClient.cs#L85

Comment (msach22) [inline OpenTokTest/OpenTokTest.cs]: I will do a separate PR in the future to have better error handling for all requests!

File: OpenTokTest/OpenTokTest.cs [hunk 3]
```diff
@@ -662,6 +662,32 @@ public void DeleteArchiveTest()
...
+        {
+            string sessionId = "SESSIONID";
+            string streamId = "be8f21f4-a133-43ae-a20a-bb29a1caaa46";
+            string returnString = "{\n" +
+                                    " \"id\" : \"" + streamId + "\",\n" +
+                                    " \"name\" : \"johndoe\",\n" +
+                                    " \"layoutClassList\" : [\"asdf\"],\n" +
```
Comment (JayTokBox) [inline OpenTokTest/OpenTokTest.cs]: sorry forgot this before
an empty layoutClassList test to revert back to default or a check if this is handled ok in code.

**Comment by msach22** [comment]
Closing in favor of #110

**Pull request closed without merging**